universal-login-wallet/test/integration/pages/Notifications.ts

```diff
@@ -5,20 +5,28 @@ export default class NotificationsPage {
   constructor(private wrapper: ReactWrapper) {}
 
   async clickConfirmButton () {
-    await waitForUI(this.wrapper, () => this.wrapper.exists('.btn-confirm'));
-    const button = this.wrapper.find('button.btn-confirm');
-    button.simulate('click');
+    await waitForUI(this.wrapper, () => this.wrapper.exists('#confirm'));
+    this.wrapper.find('#confirm').simulate('click');
     await this.waitForNotificationDisappear();
   }
 
   async clickRejectButton () {
-    const button = this.wrapper.find('button.notification-reject-btn');
-    button.simulate('click');
+    await waitForUI(this.wrapper, () => this.wrapper.exists('#reject'));
+    this.wrapper.find('#reject').simulate('click');
     await this.waitForNotificationDisappear();
   }
 
+  async inputSecurityCode(securityCode: number[]) {
+    await waitForUI(this.wrapper, () => this.wrapper.text().includes('Security code'));
+    for (const number of securityCode) {
+      const button = this.wrapper.find(`#btn-${number}`).first();
+      button.simulate('click');
+      await waitForUI(this.wrapper, () => this.wrapper.find(`img[alt="${number}"]`).length === 2);
```

Comment:
What if we have repetitions? Shouldn't we first check the number of occurrences that checked it was increased?